201-file-share-private-endpoint/azuredeploy.json

```diff
@@ -1037,37 +1034,13 @@
     }
   ],
   "outputs": {
-    "storageAccountPrivateEndpoint": {
-      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('fileStorageAccountPrivateEndpointName')), '2020-07-01', 'Full')]",
-      "type": "object"
-    },
-    "blobStorageAccountPrivateEndpoint": {
-      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('blobStorageAccountPrivateEndpointName')), '2020-07-01', 'Full')]",
-      "type": "object"
-    },
-    "fileStorageAccount": {
-      "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', parameters('fileStorageAccountName')), '2019-06-01', 'Full')]",
-      "type": "object"
-    },
-    "blobStorageAccount": {
-      "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', parameters('blobStorageAccountName')), '2019-06-01', 'Full')]",
-      "type": "object"
-    },
     "adminUsername": {
       "value": "[parameters('adminUsername')]",
       "type": "string"
     },
-    "workspaceName": {
-      "value": "[parameters('workspaceName')]",
-      "type": "string"
-    },
-    "scriptFileUri": {
-      "value": "[variables('scriptFileUri')]",
+    "adminPasswordOrKey": {

```

Comment:
Need to remove this from outputs as it may be secret.

Reply:
I will remove it, but how can a user retrieves the password when it's generated randomly from the template? In my opinion, you or someone else should specify what it could be added and what should not be added to outputs in the best practices to avoid roundtrips. I tried to facilitate user's life by printing outputs but you rejected all of them :) So I will remove all the outputs because it's unclear to me which ones are allowed and which ones are not.